cogs/annual_handover_and_reset.py

```diff
@@ -138,13 +137,13 @@ async def committee_handover(self, ctx: TeXBotApplicationContext) -> None:
             logger.debug("Giving user: %s, the committee role.", committee_elect_member)
             await committee_elect_member.add_roles(
                 committee_role,
-                reason=f"{ctx.user} used TeX-Bot slash-command: \"committee-handover\"",
+                reason=f'{ctx.user} used TeX-Bot slash-command: "committee-handover"',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes